truss-chains/truss_chains/example_chainlet.py

```diff
@@ -1,31 +1,35 @@
+import random
+
+# for more on chains, check out https://truss.baseten.co/chains/intro
 import truss_chains as chains
 
 
-class DummyGenerateData(chains.ChainletBase):
-    def run_remote(self) -> str:
-        return "abc"
+# By inhereting chains.ChainletBase, chains will know to create a chainlet that hosts the RandInt class
+class RandInt(chains.ChainletBase):
+
+    # run_remote must be implemented by all chainlets. This is the code that will be executed at inference time
+    def run_remote(self, max_value: int) -> int:
+        return random.randint(1, max_value)
+
 
+# The @chains.mark_entrypoint decorator indicates that this Chainlet is the entrypoint.
+# There must be exactly one entrypoint per chainlet
```

Comment:
Per **chain**.

More concretely per source file (the purpose of the decorator is only to simplify the deploy command to only require the specification of a "target source file", but not a "target class" within that file).

Reply:
update to specify one entrypoint per chain